8.Итоговая_работа_на_циклы/8.2.2.py

```diff
@@ -0,0 +1,15 @@
+n = 8
+count = 0
+maximum = -10**6 - 1
+
+for i in range(1, n + 1):
```

Comment:
Переменная цикла `i` не используется, значит ее нужно назвать `_`:
```suggestion
for _ in range(1, n + 1):
```